### DISCUSSION TITLE
How to preserve ICC color profiles when converting TIFFs to JPEGs

### DISCUSSION THREAD

**Discussion by Naflakusk** [ImageMagick/ImageMagick#7032] [Help]
### ImageMagick version

7.1.1-26

### Operating system, version and so on

PC

### Description

Hello all. I need to batch process a lot of TIFF images to JPEGs. The process works but strips out the ICC Color Profile in the process along with all the metadata embedded in the photos (both the camera related info and Copyright info I have added manually). I need to keep all that info in the final JPEGs I am generating. Can anyone tell me how to go about doing that? Many thanks in advance.

**Comment by snibgo** [comment]
> The process works but ...

Please show the commands you used.

A command like this...
```
magick in.tiff out.jpg
```
... should copy any embedded ICC colour profile from the input to the output.

It won't copy tiff fields such as camera info. You can use exiftool to do that.
